CHANGELOG.md

```
@@ -2,12 +2,22 @@
 
 All notable changes to this project will be documented in this file. For commit guidelines, please refer to [Standard Version](https://github.com/conventional-changelog/standard-version).
 
+## v1.5.4
+
+ **Notes**:
+ - pwa installation name now capped at 30 characters instead of 12 (#2699)
+
+ **BugFixes**:
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)